crates/mithril-oxide-sys/cpp/IR/Value.hpp

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <memory>
+
+#include <mlir/IR/Value.h>
+#include <rust/cxx.h>
```

Comment:
Assuming `BlockArgument` is defined in `mlir/IR/Block.h`, then:
```suggestion
#include <mlir/IR/Block.h>
#include <mlir/IR/Value.h>
#include <rust/cxx.h>
```